extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -252,9 +263,36 @@ class QueueProcessor extends EventEmitter {
      */
     _resumeService() {
         this._consumer.resume(this.site);
+        this._updateZkServiceStatus(this._consumer.getServiceStatus());
         this.logger.info(`resumed replication for location: ${this.site}`);
     }
 
+    _getStatusZkPath() {
+        return `${zookeeperReplicationNamespace}${ZK_CRR_STATUS_PATH}/` +
+            `${this.site}`;
+    }
+
+    /**
+     * Update Kafka consumer status in zookeeper for this site-defined
+     * QueueProcessor
+     * @param {boolean} status - true if consumer is enabled
```

Comment:
Better store actual meaningful status names instead of using booleans that only have implicit meaning (in the sense that `true` is not a meaningful status for an observer).

For example, `active` / `paused` would work. Or we can also store a JSON object like `{ "paused": true/false }`, which works better to later extend the status with more info.

Reply:
For scheduled resume, I realized it would probably be best to hold status/state for a queueprocessor at a common node, so I think using a JSON object would work best. I'll reflect the changes to a JSON object in this PR. Following this model:
`{ "paused": true/false, "scheduledResume": <DateString> }` or if you have a better format, please let me know and I can reflect them in the relevant commits

Reply:
I like the idea that the scheduled resume is set in zookeeper, so that we do not rely on orbit to re-enable replication at the requested time.